src/rfpimp.py

```diff
@@ -1,4 +1,10 @@
 """
+Attention! This is a slight customized version of rfpimp. The official
```

Comment:
if we've fixed the backward compat, do we need this comment?

Reply:
actually, i'll fix after merge. thanks!

Reply:
About the comment. I added it for the file that I am using in my project just to nor forget to delete it later. I shouldn't have pushed it, in the following commit I removed it.